spec/features/workflow/schedule_spec.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+RSpec.feature "Scheduling an edition" do
+  scenario do
+    given_there_is_an_edition
+    when_i_visit_the_summary_page
+    and_i_click_on_schedule_publishing
+    then_i_see_a_default_date_and_time_selected
+
+    when_i_choose_a_scheduled_date_and_time
+    and_i_click_save
+    then_i_see_the_edition_has_a_set_scheduled_publishing_date_and_time
+  end
+
+  def given_there_is_an_edition
+    @edition = create(:edition)
+  end
+
+  def when_i_visit_the_summary_page
+    visit document_path(@edition.document)
+  end
+
+  def and_i_click_on_schedule_publishing
+    find(".govuk-details__summary-text", text: "Schedule publishing").click
+  end
+
+  def then_i_see_a_default_date_and_time_selected
+    default_date = Time.zone.tomorrow
+    expect(page).to have_selector(
```

Comment:
Another minor one, but I think we can do this with `find_field` and `.value`, which should make the expectation a little easier to read :-).